site/docs/components/carousel/usage.mdx

```diff
@@ -8,6 +8,48 @@ data:
   $ref: ./#/data
 ---
 
+## How to build
+
+Carousel is a composable pattern of Salt components that uses the [Embla](https://www.embla-carousel.com/) API.
+
+A basic carousel navigates through slides individually, one at a time.
+
+<LivePreview componentName="carousel" exampleName="Basic" />
+
+<br />
+Adding a title serves as a heading for the carousel group.
+
+<LivePreview componentName="carousel" exampleName="Title" />
+<br />
+`CarouselTabList` is an aria `tablist` providing support for screen readers to
+announce the slides as the associated`tab`. To enhance accessibility, ensure
+that each slide has a unique id. This will facilitate the association between
+the `tablist` and `tab`, thereby improving screen reader functionality.
+
+A `tablist` provides a visual indicator of the current slide and includes clickable elements for direct navigation to specific slides, along with keyboard navigation controls. Ensure the tablist is positioned at the top or bottom of the carousel, alongside other controls like `CarouselNextButton` and `CarouselPreviousButton`.
+
+<LivePreview componentName="carousel" exampleName="TabList" />
+
+<br />
+Additional actions, such as[`Link`](../link), can be incorporated into a
+carousel slide, but avoid embedding other clickable actions during animation to
+prevent accidental clicks while slides are transitioning.
+
+`CarouselCard` also supports actions, through the `actions` prop.
+
+Content within slides can be tabbable but ensure that off-screen content is not tabbable, as this can lead to a poor user experience.
+This example ensures that CSS is applied to the off-screen content, preventing it from being tabbable.
+
+<LivePreview componentName="carousel" exampleName="CardActions" />
```

Comment:
Focus is lost when using arrow keys after you focus on the actions

Reply:
this is fixed